web/pages/home/index.tsx

```diff
@@ -200,17 +210,31 @@ function renderSections(
     score: CPMMBinaryContract[]
   }
 ) {
+  const HOME_SECTIONS_IDS = HOME_SECTIONS.map((s) => s.id)
+  type sectionTypes =
```

Comment:
I don't like having to duplicate this from HOME_SECTION_IDS, but the alternative was adding "featured" to sectionContracts for now reason (as it doesn't make conceptual sense). Open to other options if you can think of any @jahooma

Reply:
Nice!